### PR TITLE
Practicum Sequence Change (Back to Mandiri) and 'About' Update for 22/23 Period

### PR DESCRIPTION
**Pull request by Soedipo**
1. I reverts the Mandiri from Presentation because practicum will be onsite.
It changes from :
**TA -> PRESENTATION -> JURNAL -> TK**
to
**TA -> MANDIRI -> JURNAL -> TK**

2. I updates Biomedic Engineering in 'about' and gajiAsisten for 22/23

### CODE REVIEW

File: app/Http/Controllers/AsistenController.php [hunk 1]
```diff
@@ -119,8 +119,8 @@ public function show($asisten_id)
         if(count($allAsistenData) !== 0){
 
             $ratingAsisten /= count($allAsistenData);
-            $gajiAsisten = $regHistoryCount * 25000;
-            $gajiAsisten += $intHistoryCount * 62500;
+            $gajiAsisten = $regHistoryCount * 31250;
```
Comment (IritaSee) [inline app/Http/Controllers/AsistenController.php:122]: We should modify how we count `gaji` in the future XD

File: resources/js/Pages/Praktikan.vue [hunk 2]
```diff
@@ -411,15 +411,17 @@
...
+                              Lanjut Ke Jurnal
                             </div>
                           </div>
                         </div>
                       </div>
                     </div>
                   </div>
                 </div>
-                <div class="w-1/4 h-full flex-row overflow-y-hidden">
+                <div class="w-1/4 h-full flex-row overflow-y-hidden"
```
Comment (IritaSee) [inline resources/js/Pages/Praktikan.vue:422]: A bit forget about the frontend of practicum sequence, can you check later? @adhiiimf

File: resources/js/Pages/Praktikum.vue [hunk 3]
```diff
@@ -592,14 +592,14 @@
                 TA
               </span> 
               <span class="font-overpass-bold text-9xl " 
-                  :class="[{ 'visible': statusPraktikum === 2 },
-                          { 'hidden': statusPraktikum != 2 }]">
-                PRESENTASI
+                  :class="[{ 'visible': statusPraktikum === 2 || statusPraktikum === 123 },
```
Comment (IritaSee) [inline resources/js/Pages/Praktikum.vue:595]: Reminder: status `123` is for `runmod` purposes, look up into previous periods `runmod` rundown!

**Review by IritaSee — APPROVED** [review]
LGTM, but i'd prefer @adhiiimf to look up first, since his memory about the web's scheme is still fresh XD

File: resources/js/Pages/Praktikan.vue (hunk 2 above)
Comment (Soedipo) [reply resources/js/Pages/Praktikan.vue:422]: For the frontend of practicum sequence is:
1. Intro
start
2. TA start
TA
3. After TA (show the score)
score TA
4. Jurnal
jurnal

6. Mandiri
mandiri
8. TK
TK
9. After TK (show score)
TKscore
10. feedback
-->feedback

Comment (adhiiimf) [reply resources/js/Pages/Praktikan.vue:422]: > A bit forget about the frontend of practicum sequence, can you check later? @adhiiimf

sorry for late respond, okey bang i'll check this

Comment (adhiiimf) [reply resources/js/Pages/Praktikan.vue:422]: > For the frontend of practicum sequence is:
> 
> 1. Intro
>    start
> 2. TA start
>    TA
> 3. After TA (show the score)
>    score TA
> 4. Jurnal
>    jurnal
> 5. Mandiri
>    mandiri
> 6. TK
>    TK
> 7. After TK (show score)
>    TKscore
> 8. feedback
>    -->feedback

okayy good, that's right wwkwk good job dip

File: resources/js/Pages/Praktikum.vue (hunk 3 above)
Comment (adhiiimf) [reply resources/js/Pages/Praktikum.vue:595]: @Soedipo so you add this with purpose if runmod mode practicum then praktikan can access TA section? that's right?

**Review by adhiiimf — COMMENTED** [review]
Okay this is very good from me, but before i approve this `PR`, i want to know the answer that i ask to you wkwkw, thankyou @Soedipo

File: resources/js/Pages/Praktikum.vue (hunk 3 above)
Comment (Soedipo) [reply resources/js/Pages/Praktikum.vue:595]: yes, but TA and TK section will be skipped by PJ when runmod, is it ok?

File: app/Http/Controllers/AsistenController.php (hunk 1 above)
Comment (fakhrip) [reply app/Http/Controllers/AsistenController.php:122]: Yeah this is correct, it should not be hardcoded all the time actually 😅

File: resources/js/Pages/Praktikan.vue [hunk 4]
```diff
@@ -1174,7 +1177,7 @@ export default {
       ],
 
       badScoreText: [
-        "Makanya sebelum praktikum belajar dulu ya",
+        "Ayo lebih semangat lagi belajarnya!!!",
```
Comment (fakhrip) [inline resources/js/Pages/Praktikan.vue:1180]: I love this change 😄 , sorry about the passive agressive statement before xD

File: resources/js/Pages/Praktikum.vue (hunk 3 above)
Comment (adhiiimf) [reply resources/js/Pages/Praktikum.vue:595]: okeyy, good

> yes, but TA and TK section will be skipped by PJ when runmod, is it ok?

**Review by adhiiimf — APPROVED** [review]
Sorry for the slow respond from mee wkwkw, okay that's good dippp

**Pull request merged**